modular_skyrat/modules/organs/code/organs.dm

```diff
@@ -0,0 +1,11 @@
+/// Copy traits from one organ to another - e.g. with custom roundstart organs that should still get species traits applied.
+/obj/item/organ/proc/copy_traits_from(obj/item/organ/old_organ)
+	if(isnull(old_organ))
+		return
+
+	// make sure the organ gets the actions from any species specific things (like hemophage blood drain, etc)
+	for(var/datum/action/action as anything in old_organ.actions)
+		add_item_action(action.type)
+	// copy corruption flag from hemophage if applicable
+	if(old_organ.organ_flags & ORGAN_TUMOR_CORRUPTED)
+		organ_flags |= ORGAN_TUMOR_CORRUPTED
```

Comment:
This shouldn't be needed, and will only conflict with my Hemophage refactor that's currently up, addressing this issue in particular.

Feel free to give it a glance and leave a review if you feel like it, however. https://github.com/Skyrat-SS13/Skyrat-tg/pull/22625